components/locale/Switcher.vue

```diff
@@ -0,0 +1,41 @@
+<script setup>
+  const { locale, locales } = useI18n()
+  // generates URLs to switch between locales
+  const switchLocalePath = useSwitchLocalePath()
+  const availableLocales = computed(() => {
+    // returns a list of locales, exluding current locale
+    return locales.value.filter((i) => i.code !== locale.value)
+  })
+  const currentLocale = computed(() => {
+    return locales.value.filter((i) => i.code === locale.value)
+  })
+</script>
+
+<template>
+  <VMenu transition="scale-transition">
+    <!-- activator btn -->
+    <template #activator="{ props }">
+      <VBtn
+        v-bind="props"
+        rounded="lg"
+        prepend-icon="mdi-translate"
+        append-icon="mdi-chevron-down"
+        variant="text"
+      >
+        <!-- dependent on current locale-->
+        {{ currentLocale[0].code.toUpperCase() }}
+      </VBtn>
+    </template>
+
+    <VCard class="d-flex flex-column mt-2 pa-2" rounded="lg" :elevation="3">
+      <!-- buttons for locales -->
+      <VBtn
```

Comment:
Change the button variant to `text`. Should also use a flag or something for prepended icon. The text NO and EN might not be descriptive enough.

Reply:
Should be fixed

Reply:
should be fixed